optimum_benchmark/backends/base.py

```diff
@@ -68,7 +68,12 @@ def __init__(self, config: BackendConfigT):
             self.automodel_loader = get_timm_automodel_loader()
             self.pretrained_processor = None
             self.generation_config = None
-
+        elif self.config.library == "llama_cpp":
+            self.logger.info("\t+ Benchmarking a Llama.cpp model")
+            self.pretrained_config = get_transformers_generation_config(self.config.model, **self.config.model_kwargs)
```

Comment:
```suggestion
            self.generation_config = get_transformers_generation_config(self.config.model, **self.config.model_kwargs)
```

Reply:
if this what you meant

Reply:
I don't think the generation config is used anywhere by the current implementation of llama cpp

Reply:
Indeed, it is not needed, so i removed it